ops/docker/grafana/provisioning/dashboards/geth.json

```diff
@@ -0,0 +1,19870 @@
+{
```

Comment:
Does this file need to be committed?

Reply:
Not a lot of great choices as this dashboard is coming from an external source.

What I could come up with:

Have the user log into grafana and import the dashboard from https://grafana.com/grafana/dashboards/13877
Have another container fetch, template and save the file to the provisioning path
Have another container fetch, template and POST it to the grafana api

Reply:
Removed the dashboard from the repo in favor of a script to pull in a list of dashboards and share them with grafana though a Docker volume.

This list is just in the script for simplicity, and includes the dashboard link @tynes referenced.